### ISSUE TITLE
[EN][Item] No'Kaled, the Elements of Death 

### ISSUE THREAD

**Issue by Psychobird7** [Atlantiss/BugTracker#10042]
**Description**: No'Kaled's proc rates are incorrect. -- damage values seem correct. 
Reposting this since the other report got spammed with bullshit. 
**TLDR: Attacks performed with the MH No'Kaled seem to proc the OH No'Kaled, and vice versa, making the proc rate while wielding two No'Kaleds effectively 4 times higher than with a single No'Kaled. 
They shouldn't.  /TLDR**
**Edit: This bug also affects Gurthalak when it is being dual-wielded.** 

**Current behaviour**: 
As tested on PTR with Heroic No'Kaled(s). 
Two No'Kaleds: 1298 Procs, 10036 Melee swings. --> 12.9% proc rate. 
One No'Kaled in MH, shield in OH: 348 Procs, 5022 Melee swings. --> 6.9% proc rate. 
You can see having two No'Kaleds equipped doubles the proc rate, which it should not. 
**Edit: This bug also affects Gurthalak when it is being dual-wielded.** 

**Expected behaviour**: 
The proc chance should be 6.5% according to Wowhead and Simcraft. 
This is correct, **if using one No'Kaled**. 

The proc rate should remain the same with two No'Kaleds equipped. 
See the following code taken from Simcraft. https://pastebin.com/NNFb5dAz 
"// Only attacks from the slot that have No'Kaled can trigger it
  // unless the action ignores slot" 
Basically, attacks performed with the MH No'Kaled seem to proc the OH No'Kaled, and vice versa. They shouldn't. 

**Steps to reproduce the problem**:

1. Go on PTR, equip No'Kaled(s) 
2. Hit a dummy 
3. Check the proc rates

**Revision**: 5769 (5763 on PTR)

**Comment by Psychobird7** [comment]
To all the retards who were spamming on my other report. 

Bring some **evidence** before you post, please.

**Comment by fluessig** [comment]
imo u should have pointed out what u meant more clearly
i misunderstood what u meant until i had searched for proof on nokaled profiting from being dual wielded

what you are reporting here: dual wield nokaled > quadrouple the proc rate of a single nokaled equipped
what people see in this report: dual wield nokaled > double proc rate of single nokaled equipped

"Basically, attacks performed with the MH No'Kaled proc the OH No'Kaled, and vice versa. They shouldn't."
that sentence or a variation of it shouldve been the headline, and wouldve saved a lot on confusion

**Comment by Psychobird7** [comment]
Yeah, because clearly, expecting people to read the *full* report is too much, right?

**Comment by fluessig** [comment]
you may have a point, then again filling the report with unecessary and complicating things doesnt help your cause at all
keep it plain and simple and get to the point
im not trying to bash u or anything,. just thought id point out that people misunderstand this report easily (as i did myself)

**Comment by Psychobird7** [comment]
I know what you mean, but I did use simple English :P  
edited the OP for clarity.

**Comment by fluessig** [comment]
tyvm

**Comment by Psychobird7** [comment]
The answer to your first question is under "Current behaviour".  
"Two No'Kaleds: 1298 Procs, 10036 Melee swings. --> 12.9% proc rate." 
10036 swings with two No'Kaleds (5018 with each hand) lead to a proc rate much higher than it should be. 
5018 swings in MH with a 6.5% proc chance, and 5018 swings in OH with a 6.5% proc chance, should lead to: 
5018\*0.065+5018\*0.065=652.34 procs. 
And 10036*0.065 is the *exact same*. -- the *actual* number of procs would be slightly lower, because OH attacks miss more, hence creating fewer procs. 
Currently, it leads to ~1300 procs. 

A No'Kaled in OH should obviously still proc, but it should not double both weapons' proc rates. 

What you're describing with Landslide is intended, however an attack performed with your MH should (it's possible that this does happen on atlantiss, but it *should*) *not* proc a Landslide enchant on your OH. 

I still don't *really* understand how it was that hard to understand, there was no complex math, and I specifically used 5000 and 10000 swing tests to make it clearer. (or so I thought) 

Either way, glad it's cleared up now.
